.github/workflows/post-submit.yaml

```diff
@@ -3,32 +3,31 @@ on:
   push:
     branches:
       - main
-      - 'release-*'
     tags:
       - 'v*'
-  release:
-    types:
-      - created
 
 jobs:
-  push_to_registry:
-    name: Push a container image to quay.io/medik8s
+  push-images:
+    name: Build and push images to quay.io/medik8s
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          
+
       - name: Log in to Quay.io
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
           registry: quay.io
 
-      - name: Prepare Container Image
-        run: make docker-build
+      - name: Build and push CSV 0.0.1 + latest images, for pushes
```

Comment:
I don't see where that 0.0.1 will be set

Reply:
IIUC it is now set in Makefile,